src/utils/logError.ts

```diff
@@ -0,0 +1,5 @@
+import { captureException } from "@sentry/browser";
+
+export default function logError(err: any) {
+  captureException(err);
```

Comment:
In development, you don't want to log to sentry, you maybe want to log to the console. You can check the environment here.

Reply:
ok, I'll look into that

Reply:
I just finished working on this